src/BursarExportPlugin.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  Button,
+  LoadingPane,
+  Pane,
+  PaneFooter,
+} from '@folio/stripes/components';
+import { useStripes } from '@folio/stripes/core';
+import { FormApi } from 'final-form';
+import React, { useCallback, useRef } from 'react';
+import formValuesToDto from './api/dto/to/formValuesToDto';
+import schedulingToDto from './api/dto/to/schedulingToDto';
+import useAutomaticSchedulerMutation from './api/mutators/useAutomaticSchedulerMutation';
+import useManualSchedulerMutation from './api/mutators/useManualSchedulerMutation';
+import ConfigurationForm, { FORM_ID } from './form/ConfigurationForm';
+import useInitialValues from './hooks/useInitialValues';
+import FormValues from './types/FormValues';
+import { FormattedMessage } from 'react-intl';
+
+export default function BursarExportPlugin() {
+  const stripes = useStripes();
+
+  const manualScheduler = useManualSchedulerMutation();
+  const automaticScheduler = useAutomaticSchedulerMutation();
+
+  const formApiRef = useRef<FormApi<FormValues>>(null);
+
+  const submitCallback = useCallback(async (values: FormValues) => {
+    if (values.buttonClicked === 'manual') {
+      await manualScheduler(formValuesToDto(values));
+    } else {
+      await automaticScheduler({
+        bursar: formValuesToDto(values),
+        scheduling: schedulingToDto(values.scheduling),
+      });
+    }
+  }, []);
+
+  const initialValues = useInitialValues();
+
+  if (initialValues === null) {
+    return (
+      <LoadingPane
+        paneTitle={
+          <FormattedMessage id="ui-plugin-bursar-export.bursarExports.paneTitle" />
+        }
+        defaultWidth="fill"
+        footer={
+          <PaneFooter
+            renderStart={
+              <Button disabled>
+                <FormattedMessage id="ui-plugin-bursar-export.bursarExports.button.runManually" />
+              </Button>
+            }
+            renderEnd={
+              <Button disabled buttonStyle="primary">
+                <FormattedMessage id="ui-plugin-bursar-export.bursarExports.button.save" />
+              </Button>
+            }
+          />
+        }
+      />
+    );
+  }
+
+  return (
+    <Pane
+      defaultWidth="fill"
+      footer={
+        <PaneFooter
```

Comment:
this is almost a code duplication with loader pane

you can do 

```
const footer = <PaneFooter...
```

above to make code cleaner - or create new component

Reply:
I think that I would be a lot messier to add the and/or logic below rather than to just have two separate returns.

Reply:
you can always create new component if you don't want to put higher, but it's just the same code, and having 2 separate shouldn't be a choice when possible